### PR TITLE
Add support for adding assets to AUR repository

### LINKED ISSUE #20: Support pushing other files to AUR repository
I would like to publish other files (for example, `.gitignore`, `.install` script, patches, other assets...) to the AUR repository along with the `PKGBUILD` and `.SRCINFO`. I think this could be done by adding another input to the action (maybe `assets`) that accepts a list of files (maybe also whole directories or glob patterns), copies them to `/tmp/local-repo` and commits them along with existing files.

### PR DESCRIPTION
**Pull request by filips123**
This implements support for adding assets to AUR repository and closes #20.

I added `assets` key which can contain glob patterns of additional files that are copied to repo. Globs are expanded by bash and I also added `GLOBIGNORE=".:.."` to prevent copying parent directory (`..`) when trying to match hidden files.

I created a test repo that uses this, but with all actual publishing [disabled](https://github.com/filips123/github-actions-deploy-aur/commit/20cba93ba22a58ca8a8be799037bb52d1fd21e2d), and it [seems to work](https://github.com/filips123/github-actions-deploy-aur-test/runs/3242122088).

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -51,20 +57,35 @@ echo '::group::Cloning AUR package into /tmp/local-repo'
...
-makepkg --printsrcinfo >.SRCINFO
+echo '::group::Copying files into /tmp/local-repo'
+{
+  echo "Copying $pkgbuild"
+  cp -r "$pkgbuild" /tmp/local-repo/
+}
+# shellcheck disable=SC2086
+# Ignore quote rule because we need to expand glob patterns to copy $assets
+{
+  echo "Copying " $assets
```
Comment (KSXGitHub) [inline build.sh:68]: You can remove trailing whitespace after `Copying`.

**Comment by KSXGitHub** [comment]
I will merge this for now, but without creating a tag. You may try it out in your workflow by replacing `github-actions-deploy-aur@<TAG>` with `github-actions-deploy-aur@beeaa2c21fccb825b1514089c02f260001c697bd`. Please tell me the result so that I may make an official release.

**Comment by filips123** [comment]
Thanks. I won't test it yet as I still have to do a few things with my project, but it will probably be ready this or next week.

**Comment by ReenigneArcher** [comment]
Has this been tested? If so could a new tag/release be made?

**Comment by KSXGitHub** [comment]
Some is attempting to use this feature: https://github.com/KSXGitHub/github-actions-deploy-aur/issues/28

**Comment by KSXGitHub** [comment]
v2.3.0 has been released.

**Pull request merged**